plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -66,12 +73,13 @@ const useStyles = makeStyles(theme => ({
  */
 export type StepperProps = {
   manifest: TemplateParameterSchema;
-  extensions: NextFieldExtensionOptions<any, any>[];
+  extensions: FieldExtensionOptions<any, any>[];
```

Comment:
Hmm - don't think there is to be honest. Because they're extensions of any types, I think we did try with unknown, but I think that caused other issues before.